### PR TITLE
Add cruise control

### PR DESCRIPTION
**Pull request by Baisang**

### CODE REVIEW

**Comment by encadyma** [comment]
Looks nice 👍

**Comment by Baisang** [comment]
Seems to work https://i.fluffy.cc/9lLJ9ptVdWHdBznSGzjSRTwNC8KVHWRn.png

File: kubernetes/kafka.yaml.erb [hunk 1]
```diff
@@ -60,10 +60,21 @@ spec:
...
+        - |
+          cp -v /opt/cruise-control/cruise-control/build/dependant-libs/cruise-control-metrics-reporter.jar /opt/kafka/libs/extensions/cruise-control-metrics-reporter.jar
+        volumeMounts:
+        - name: extensions
+          mountPath: /opt/kafka/libs/extensions
       containers:
       - name: k8skafka
         imagePullPolicy: Always
-        image: "docker.ocf.berkeley.edu/kafka:<%= version %>"
+        image: "docker.ocf.berkeley.edu/kafka::<%= version %>"
```
Comment (jvperrin) [inline kubernetes/kafka.yaml.erb]: Why add two colons here?

Comment (Baisang) [reply kubernetes/kafka.yaml.erb]: you got me

**Pull request merged**